config/database.yml

```diff
@@ -0,0 +1,29 @@
+# for postgres comment lines below in
+# It is preferable to use postgres because this
+# is what's used in production and it is necessary
+# to work on the coldmap.
+development: &development
+  adapter: postgresql
+  database: dev_database
+  username: dave_database_user
```

Comment:
Was this line a typo or is it this way in the encrypted database.yml also? I think this might be dev_database_user instead of dave_database_user.